shared/chat/conversation/messages/react-button/container.tsx

```diff
@@ -23,6 +23,7 @@ export type OwnProps = {
 
 export type WrapperProps = {
   active: boolean
+  cannotWrite: boolean
```

Comment:
It looks like you've made `cannotWrite` mandatory everywhere. As such, I think it makes more sense to name it `canWrite`, since the check will otherwise always result in a harder-to-read double negative.

Reply:
Fair enough. I pushed a rebase + that change.